ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -311,6 +311,18 @@ const syncExternalModules = async() => {
   }
 };
 
+// exported for test
+export const getEnabledModules = (modules: AccessModule[], navigate): AccessModule[] => fp.flatMap(moduleName => {
```

Comment:
these 2 functions have the same logic as they did when they were inline (old lines 524+)